### ISSUE TITLE
Allow multiple components with same name on GoogleV3 Geocoder

### ISSUE THREAD

**Issue by pratheekrebala** [geopy/geopy#407]
The Google API allows specifying multiple components of the same type for [component filtering](https://developers.google.com/maps/documentation/geocoding/intro#ComponentFiltering)
e.g. 

`administrative_area:VA|administrative_area:Arlington` 

However, this is currently not possible using the GoogleV3 geocoder since the method requires that components be passed in as dict.

One way to add this functionality might be to also accept a list of tuples:
```
components=[
    ('administrative_area',  'VA'),
    ('administrative_area', 'Arlington')
]
```

I can make a PR if this is within scope.

**Comment by KostyaEsmukov** [comment]
Looks like a good idea to me! A PR would be welcome

**Comment by pratheekrebala** [comment]
PR in #409

**Comment by KostyaEsmukov** [comment]
Resolved with #409
